cmd/kubeadm/app/cmd/phases/kubelet.go

```diff
@@ -92,3 +123,128 @@ func runKubeletStart(c workflow.RunData) error {
 
 	return nil
 }
+
+// NewKubeletStartJoinPhase creates a kubeadm workflow phase that start kubelet on a node.
+func NewKubeletStartJoinPhase() workflow.Phase {
```

Comment:
Once this is is in a separated file under `phases\join` this name can be simplified into `NewKubeletStartPhase `